### ISSUE TITLE
Build error starting version 0.13.0

### ISSUE THREAD

**Issue by mauron85** [react-toolbox/react-toolbox#179]
Starting version 0.13.0 webpack is enable to build my app. Build fails with following error:

```
ERROR in ./~/react-toolbox/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./list/ListSubHeader in /Users/finch/dev/mapilary/mapilary-widget-ng/node_modules/react-toolbox/lib
 @ ./~/react-toolbox/lib/index.js 145:25-56
```

Version 0.12.8 is fine.

In my app, imports are defined as:
```
import { AppBar, Input, Button, Navigation } from 'react-toolbox'
```

**Comment by javivelasco** [comment]
I'm looking at it in a couple of hours, thanks!

**Comment by wrench7** [comment]
I've also had to change the way I was importing react-toolbox.

Before the 0.13.0 update I was using
```import rt from 'react-toolbox';``` and the in my render function using ```<rt.Button/> ... <rt.FontIcon/> ... etc``` to actually instantiate the components. This allows me very easily to use any of the components without having to update the import statement and it also nicely namespaces the react-toolbox components from the ones I'm developing.

I've been using Material UI before coming to react-toolbox, and the way they export their modules allows better IDE introspection and also allows the previous aliasing approach. Happy to open up a PR if you're open to suggestions in this area?

**Comment by javivelasco** [comment]
I'm particularly always open to discuss new ways to do stuff :) There is something you have to take into account when you import the whole library that way: each component references it's own SASS file and if you import everything, the final build will include the whole CSS even if you are not actually using it. That's why we recommend to import just the module you are using.

What I don't get is why did you have to change the way you are importing. We are still exposing all the components in the `/lib/index.js` file so it should work. Are you getting any errors??

**Comment by wrench7** [comment]
I'll have to test out how things can get bloated if you import the index file (my understanding was that webpack was intelligent enough to not do that, but I may be wrong).

As for why I had to change the way I was importing, looking at the index.js file in the components folder in  v0.12.13 compare to v0.13.1 you'll see that there's no default export in that file anymore ;-)

**Comment by wrench7** [comment]
Just a heads up, I can use ```import * as rt from 'react-toolbox';``` but am still interested to see if the removal of the exports from the index.js was intentional.

**Comment by javivelasco** [comment]
I have to take a deeper look at it but for now I can tell you it's probably missed.

**Comment by nathanmarks** [comment]
@javivelasco It's because everything is now a named export, rather than importing everything and then exporting a giant default object. This is my fault -- I also changed the docs to do `import * as ....`.

I could be wrong, but I believe that for proper ES6 usage once the spec is native, named exports is the way to go:

> **Why do we need named exports?**
>
> You may be wondering – why do we need named exports if we could simply default-export objects (like CommonJS)? The answer is that you can’t enforce a static structure via objects and lose all of the associated advantages (described in the next section).
(http://www.2ality.com/2014/09/es6-modules-final.html)

**Comment by nathanmarks** [comment]
@wrench7 see above

**Comment by javivelasco** [comment]
Agree with @nathanmarks, thanks for clarifying! :+1:

**Comment by wrench7** [comment]
Thanks for the clarification @nathanmarks :+1: 

Can you confirm that using imports like that will also bloat the file size as @javivelasco suggested? I'm seeing that importing components individually still results in webpack creating styles for all the Toolbox components when viewing my app in a browser. That could be because I'm using HMR, but wondered if others are seeing the same thing?

**Comment by nathanmarks** [comment]
@wrench7 So what you're saying is -- if you try to do something such as:

```
import Button from 'react-toolbox/lib/button';
```

You get the styles for every component? If so, that's obviously not good.

However...

If you `import * as Components from 'react-toolbox'`, I'm fairly certain you're going to load everything. This is because until es6 modules are native to browsers, every single module is included if you pull from the index as the code is being required common-js style and executed. Feel free to correct me if I'm wrong on this!

**Comment by javivelasco** [comment]
Can we close then?

**Comment by mauron85** [comment]
@javivelasco My issue is not resolved.

**Comment by javivelasco** [comment]
Wow, you're right. Sorry, I mixed up with other issue.
I'll take a look on it asap @mauron85

**Comment by mauron85** [comment]
## How to reproduce:

### 1. prepare react-toolbox-example
```
$ git clone https://github.com/react-toolbox/react-toolbox-example.git
$ cd react-toolbox-example
$ git checkout ba6abd0893783011f35985b2499ccbc4b0688d7b #not necessary currently latest
$ npm install
```

### 2. change imports in ```app/index.jsx```

``` jsx
// You can also include here commons if you want with import 'react-toolbox/lib/commons';
import React from 'react';
import ReactDOM from 'react-dom';
// import ToolboxApp from 'react-toolbox/lib/app';
// import Button from 'react-toolbox/lib/button';
import { ToolboxApp, Button } from 'react-toolbox'; // THIS IS THE CHANGE
import Header from './components/header';
import style from './style';

ReactDOM.render((
  <ToolboxApp>
    <Header />
    <section className={style.content}>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris. Maecenas congue ligula ac quam viverra nec consectetur ante hendrerit. Donec et mollis dolor. Praesent et diam eget libero egestas mattis sit amet vitae augue. Nam tincidunt congue enim, ut porta lorem lacinia consectetur. Donec ut libero sed arcu vehicula ultricies a non tortor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ut gravida lorem. Ut turpis felis, pulvinar a semper sed, adipiscing id dolor. Pellentesque auctor nisi id magna consequat sagittis. Curabitur dapibus enim sit amet elit pharetra tincidunt feugiat nisl imperdiet. Ut convallis libero in urna ultrices accumsan. Donec sed odio eros. Donec viverra mi quis quam pulvinar at malesuada arcu rhoncus. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In rutrum accumsan ultricies. Mauris vitae nisi at sem facilisis semper ac in est.
    </section>
    <Button label='test' />
  </ToolboxApp>
), document.getElementById('app'));

```

### 3. start dev server
```
npm start
```

### There will be errors in teminal
```
ERROR in ./~/react-toolbox/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./list/ListSubHeader in /Users/finch/dev/react-toolbox-example/node_modules/react-toolbox/lib
 @ ./~/react-toolbox/lib/index.js 135:25-56

ERROR in ./~/react-toolbox/lib/utils/polyfills.js
Module not found: Error: Cannot resolve module 'core-js/fn/array/from' in /Users/finch/dev/react-toolbox-example/node_modules/react-toolbox/lib/utils
 @ ./~/react-toolbox/lib/utils/polyfills.js 3:0-32

ERROR in ./~/react-toolbox/lib/utils/polyfills.js
Module not found: Error: Cannot resolve module 'core-js/fn/array/iterator' in /Users/finch/dev/react-toolbox-example/node_modules/react-toolbox/lib/utils
 @ ./~/react-toolbox/lib/utils/polyfills.js 5:0-36

ERROR in ./~/react-toolbox/lib/utils/polyfills.js
Module not found: Error: Cannot resolve module 'core-js/fn/map' in /Users/finch/dev/react-toolbox-example/node_modules/react-toolbox/lib/utils
 @ ./~/react-toolbox/lib/utils/polyfills.js 7:0-25

ERROR in ./~/react-toolbox/lib/utils/polyfills.js
Module not found: Error: Cannot resolve module 'core-js/fn/string/starts-with' in /Users/finch/dev/react-toolbox-example/node_modules/react-toolbox/lib/utils
 @ ./~/react-toolbox/lib/utils/polyfills.js 9:0-40

ERROR in ./~/react-toolbox/lib/utils/polyfills.js
Module not found: Error: Cannot resolve module 'core-js/fn/symbol' in /Users/finch/dev/react-toolbox-example/node_modules/react-toolbox/lib/utils
 @ ./~/react-toolbox/lib/utils/polyfills.js 11:0-28
```

**Comment by javivelasco** [comment]
Thanks for the extended info!

**Comment by javivelasco** [comment]
Fixes #179
